tests/system/View/ParserTest.php

```diff
@@ -764,7 +764,7 @@ public function testParserPluginClosure()
 			return 'Hello, ' . trim($params[0]);
 		};
 
-		$this->parser = new Parser($config, $this->viewsDir, $this->loader);
+		$this->parser             = new Parser($config, $this->viewsDir, $this->loader);
```

Comment:
this shouldn't align

Reply:
it is changed by coding standard, I need to manually disable cs : 

```bash
Running Code Sniffer...
F 1 / 1 (100%)



PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
...www/CodeIgniter4/tests/system/View/ParserTest.php  1      96
----------------------------------------------------------------------
A TOTAL OF 1 ERROR WERE FIXED IN 1 FILE
```

I've updated it.